src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -67,11 +68,19 @@ final class ValidatorPropertyMetadataFactory implements PropertyMetadataFactoryI
 
     private $decorated;
     private $validatorMetadataFactory;
+    /**
+     * @var iterable<PropertySchemaRestrictionMetadataInterface>
+     */
+    private $restrictionsMetadata;
 
-    public function __construct(ValidatorMetadataFactoryInterface $validatorMetadataFactory, PropertyMetadataFactoryInterface $decorated)
-    {
+    public function __construct(
+        ValidatorMetadataFactoryInterface $validatorMetadataFactory,
```

Comment:
We put every parameters on the same line (same everywhere).

Reply:
fixed